### DISCUSSION TITLE
Q3Q4 2022 AppFlowy's 6-month Technical Roadmap

### DISCUSSION THREAD

**Discussion by annieappflowy** [AppFlowy-IO/AppFlowy#565] [General]
![appflowy_july_dec_roadmap](https://user-images.githubusercontent.com/12026239/180709306-e1a42453-30d9-4582-a275-36f1a85da8c2.png)

Our new roadmap for July - December 2022. 
Open to feedback and participation.

**Comment by rileyhawk1417** [comment]
Looks interesting hope we can achieve everything before end of the year ^_^

**Reply by heyakyra** [reply]
Looks like things are a bit behind which is expected in the world of software 😅 I wonder if we can see an updated roadmap to v0.1.0 with the collaboration server

**Comment by lyckligtax** [comment]
Yeah, cannot wait for the http server 🥳

Will there be meetings one can join? Would like to collaborate on that part :-)

**Reply by appflowy** [reply]
> Yeah, cannot wait for the http server 🥳
> 
> Will there be meetings one can join? Would like to collaborate on that part :-)

We will share the design doc on GitHub, similar to how we share the roadmap, to get feedback. Welcome participation :D

**Reply by heyakyra** [reply]
It will be AGPL too, right?

**Reply by annieappflowy** [reply]
Yes, it will stay under AGPL; however, we plan to relicense appflowy_editor and appflowy_board to MPL.

**Comment by vip20** [comment]
Can there be sync to dav server? Letting the user to select where to save the data.

**Reply by librafrog** [reply]
Also excited about this project. I think it might just be a great alternative to apps like Notion or Obsidian
I'm not a coding expert, still I do mostly non coding tasks like fixing typos, writing documentation + translatation and some basic html, markdown tasks. **v.0.0.6** will be great if __devs__ can implement the "Grid" option to be some kind of dynamic databases like in Notion. All will check back soon to see milestones, feel free to assign me to *general help wanted* or *general technical help* Cheers 🍺

**Reply by annieappflowy** [reply]
@vip20 it's planned to start the development at v0.0.9

**Reply by annieappflowy** [reply]
@librafrog big thanks for offering the help. I'll loop you in when relevant tasks pop up :D  Please also feel free to step in

**Comment by annieappflowy** [comment]
![Q3Q4Roadmap](https://user-images.githubusercontent.com/12026239/212692480-1d47e84a-bb27-48b5-81ad-07d252401655.png)
Thanks all for keeping an eye out for the Q3-Q4 roadmap. I marked tasks we completed. There will be a new post about the Q1-Q2 2023 roadmap. Look forward to getting more exciting work done!

**Reply by heyakyra** [reply]
Thanks! Why is the grid's version history proprietary? I hope this doesn't mean AppFlowy will be going open core

x-post: https://github.com/AppFlowy-IO/AppFlowy/discussions/1715#discussioncomment-5315703
